### PR TITLE
Feat/header and footer

### PR DESCRIPTION
**Pull request by Nick0603**
# Description

Create Header and Footer

image
image
image

## Type of change
- [x] New feature (non-breaking change which adds functionality)

# How Has This Been Tested?

[x] redirect
- After Clicking 「關於茲種」in header, you will redirect to about page.
- After Clicking logo image in header, you will redirect to homepage.

test on stage: http://stage.itseed.tw/

# Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- N/A I have made corresponding changes to the documentation
- N/A My changes generate no new warnings
- N/A I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes
- N/A Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

File: components/organisms/Header/Header.desktop.module.scss [hunk 1]
```diff
@@ -0,0 +1,109 @@
...
+.header {
+  display: block;
+  width: 100%;
+  backdrop-filter: blur(16px);
+  background-color: rgba(241, 242, 244, 0.8);
+
+  @include mobile {
+    display: none;
+  }
+}
```
Comment (AlanSyue) [inline components/organisms/Header/Header.desktop.module.scss:17]: break line

File: components/organisms/Footer/Footer.tsx [hunk 2]
```diff
@@ -1,9 +1,52 @@
...
+          <span>主辦單位：新北市電腦商業同業公會</span>
+          <Image
+            alt="NTCA"
+            src="/images/NTCA-logo.svg"
+            width="46"
+            height="11"
+          />
+        </div>
+        <span className={styles.copywritingRow}>
+          Copyright © 2021 - 資訊種子培訓計畫
```
Comment (AlanSyue) [inline components/organisms/Footer/Footer.tsx:63]: Does the year `2021` need to be handled in env variable?

File: components/organisms/Footer/Footer.tsx [hunk 3]
```diff
@@ -8,28 +8,46 @@ const Footer: FC = () => {
...
           </span>
           <span>
-            <Image
-              alt="mail"
-              src="/images/icons/icon-mail.svg"
-              width="24"
-              height="24"
-            />
+            <a
+              href="mailto:itseed19th@gmail.com"
```
Comment (AlanSyue) [inline components/organisms/Footer/Footer.tsx:40]: Use `itseedsystem@gmail.com` ?

File: components/organisms/Footer/Footer.tsx (hunk 2 above)
Comment (Nick0603) [reply components/organisms/Footer/Footer.tsx]: I let the website make it dynamically. It will get the current year for here.

**Review by AlanSyue — APPROVED** [review]
LGTM

**Pull request merged**